thoth/metrics_exporter/jobs/solver.py

```diff
@@ -96,7 +96,7 @@ def get_unsolved_python_packages_versions(cls) -> None:
     def get_python_packages_solved_count_per_solver(cls) -> None:
         """Get number of solved Python packages per solver."""
         for solver_name in cls._OPENSHIFT.get_solver_names():
-            solver_info = cls.graph().parse_python_solver_name(solver_name)
+            solver_info = cls._OPENSHIFT.parse_python_solver_name(solver_name)
```

Comment:
Wrong rebase?

Reply:
not it is correct I think, we moved the logic from storages to common: https://github.com/thoth-station/common/blob/aa12a3635c015e15f7d8dbffe4e65706fb79ea78/thoth/common/openshift.py#L214